atlasdb-workload-server-distribution/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.palantir.sls-java-service-distribution'
 apply plugin: 'com.palantir.external-publish-dist'
-apply from: "../gradle/docker.gradle"
 apply from: "../gradle/shared.gradle"
+apply from: "../gradle/docker.gradle"
```

Comment:
This is very subtle, but very important. All of our images were `unspecified` as the gradle property hadn't been set yet. This get's set in `shared`!

Reply:
not for now: We might want to just have docker.gradle fail if it's included and the property isn't set - though definitely not for now!